runtime/doc/vim_diff.txt

```diff
@@ -38,7 +38,6 @@ centralized reference of the differences.
 - 'complete' excludes "i"
 - 'cscopeverbose' is enabled
 - 'directory' defaults to ~/.local/state/nvim/swap// (|xdg|), auto-created
-- 'display' defaults to "lastline,msgsep"
```

Comment:
"lastline" still differs from vim, no?

Also should have a "UI" section somewhere in `nvim-features` which mentions what used to be described in the `msgsep` help doc.